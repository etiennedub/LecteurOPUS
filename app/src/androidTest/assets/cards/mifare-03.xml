<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04785FAB</List><List index="1">2A444980</List><List index="2">A748E000</List><List index="3">0000000F</List><List index="4">00010001</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">DAFD6202</List><List index="8">01C6F74C</List><List index="9">770A4500</List><List index="10">65008000</List><List index="11">22285418</List><List index="12">01C6F7A4</List><List index="13">7A4A450C</List><List index="14">79000000</List><List index="15">20284D45</List></Pages></Mifare>