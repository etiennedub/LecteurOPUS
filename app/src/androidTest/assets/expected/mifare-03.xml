<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="06/12/2016" Time="16:18" Transfert="false" OperatorId="20" BusId="164" OperatorName="RTC" BusName="" /><List index="1" Date="06/12/2016" Time="15:52" Transfert="true" OperatorId="20" BusId="164" OperatorName="RTC" BusName="" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>
